.github/ISSUE_TEMPLATE/4_Feature_request.md

```diff
@@ -7,5 +7,5 @@ about: Ideas for other new features or improvements
 
 <!-- A clear and concise description of the new feature. -->
 
-<!-- Love Shields? Please consider donating $10 to sustain our activities:
+<!-- Love Shields? Please consider donating to sustain our activities:
```

Comment:
I'm less confident in this change. My theory is that naming an amount (and being ambiguous about whether it is one-off or monthly) might put people off? Lets try removing the concrete ask for an amount on the in-repo donate CTAs and just point people who are interested at OpenCollective for a menu of options.